### PR TITLE
add skypack-resolver

### PR DESCRIPTION
**Pull request by vinicius73**
Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/rollup/awesome/blob/master/.github/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Provide a Link A Repository for Your Addition

https://github.com/vinicius73/rollup-plugin-pika-resolver
https://www.npmjs.com/package/@vinicius73/rollup-plugin-skypack-resolver

### Please Describe Your Addition

Generate the bundle of your project using [Skypack CDN](https://skypack.dev/) for external dependencies.

### CODE REVIEW

**Comment by vinicius73** [comment]
Any news about that?

**Comment by shellscape** [comment]
Sorry we missed this. Please place your plugin in alphabetical order in that section and we can  merge it.

**Comment by vinicius73** [comment]
Thanks @shellscape 
Also I have changed the package name (pika resolver is skypack resolver now)

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ and external modules.
 - [node-globals](https://github.com/calvinmetcalf/rollup-plugin-node-globals) - Injects node globals.
 - [node-resolve-angular](https://github.com/OasisDigital/rollup-plugin-node-resolve-angular) - Node module resolution support for Angular 4+.
 - [ts-paths](https://github.com/bitshiftza/rollup-plugin-ts-paths) – Resolve modules from `tsconfig` paths.
+- [skypack-resolver](https://github.com/vinicius73/rollup-plugin-pika-resolver) - Generate the bundle of your project using [Skypack CDN](https://skypack.dev/) for external dependencies.
```
Comment (shellscape) [inline README.md:119]:
```suggestion
- [skypack-resolver](https://github.com/vinicius73/rollup-plugin-pika-resolver) - Use [Skypack CDN](https://skypack.dev/) for external dependencies.
```

**Comment by shellscape** [comment]
thanks!

**Pull request merged**